dask-gateway/requirements.txt

```diff
@@ -1,8 +1,8 @@
 # These are dependencies for installing the dask-gateway package.
 #
 aiohttp
-click
-dask>=2.2.0
-distributed>=2021.01.1
+click>=8.1.3
+dask>=2022.5.2
+distributed>=2022.5.2
```

Comment:
```suggestion
dask>=2022.1.0
distributed>=2022.1.0,!=2022.5.1,!=2022.5.2
```

Reply:
Btw, this discussion relates to https://github.com/dask/dask-gateway/issues/539.